CONTRIBUTING.md

```diff
@@ -38,13 +38,13 @@ These steps should get you up and started for local development setup. Please en
 
 - Install [Xcode](https://reactnative.dev/docs/environment-setup#xcode).
 
-  > **Note:**
+  > **Note**
   >
   > Sometimes it can take a very long time for Xcode to install. Check [here](https://apple.stackexchange.com/questions/427640/mac-app-store-xcode-download-stuck-at-installing) for troubleshooting.
 
-- Install an iOS simulator in Xcode: Xcode > Preferences > Components.
+- Install an iOS 13 simulator in Xcode: Xcode > Preferences > Platforms.
```

Comment:
xcode installed ios 16 by default in my machine and the storybook react native command was failing saying it cannot find "iPhone 13".

Reply:
I can't see "Components" tab in my preferences. I have "Platforms" from where I can install ios or any other simulators and I am on latest xcode.